xmtp/src/association.rs

```diff
@@ -66,17 +80,15 @@ impl Association {
         self.text.get_address()
     }
 
-    pub fn test() -> Result<Self, AssociationError> {
-        let key = SigningKey::random(&mut utils::rng());
-        let pubkey = key.verifying_key();
-        let addr = eth_address(pubkey).unwrap();
+    pub fn test(pub_key: Vec<u8>) -> Result<Self, AssociationError> {
```

Comment:
Now that we are actually verifying Contact validity on creation, some of the tests were failing because this signature wasn't actually over the pub_key bytes.